Clean up Images, Add Syntax Highlighting, Fix Spelling and Grammar (#211)
NuevoFoundation/workshops: content/english/pygame-pong/activity-5.md

---

# Activity 5

> Great progress so far! Now lets make use of a variable to control the font value. 

Reviewer: ```suggestion
Great progress so far! Now let's make use of a variable to change the font of the text in the game.
```

> In this activity, you need to apply the knowledge about variables you learnt in Python basics. 

Reviewer: ```suggestion
In this activity, you need to apply the knowledge about variables you learned in Python basics. 
```

> Find `// TODO (ACTIVITY 5)` in the code. Declare the variable `font_label`, assign it the value `'arial'`. Now take a look at this statement in code `font_name = pygame.font.match_font('arial')`. Can you figure out the place in this statement where you should use this new variable to control the font displayed on screen? 

Reviewer: ```suggestion
Find `// TODO (ACTIVITY 5)` in the code. Declare the variable `font_label`, assign it the value `'arial'`. Now take a look at this statement in the code `font_name = pygame.font.match_font('arial')`. Can you figure out the place in this statement where you should use this new variable to control the font displayed on screen? 
```
